_benchmark/quickstart.md

```diff
@@ -18,9 +18,17 @@ To perform the Quickstart steps, you'll need to fulfill the following prerequisi
 
 ## Set up an OpenSearch cluster
 
-Follow the instructions on the [OpenSearch download page](https://opensearch.org/downloads.html) to start up your cluster.  You can either use *Docker Compose* or download the appropriate distribution for your platform, extract or install it, then follow instructions.  For instance, on Linux or MacOS, you would run the `opensearch-tar-install.sh` script.  This will bring up a cluster with its endpoint at localhost:9200, and since the security plugin is enabled with basic authentication configured, it will expect SSL connections with the username "admin" and password "admin".  However, since the "localhost" address is not a unique public address, no certificate authority will issue an SSL certificate for it, so certificate checking will need to be disabled.
+If you don't already have an acitive OpenSearch cluster, you can launch a new cluster to being using OpenSearch Benchmark.
```

Comment:
`you can launch a new cluster to being using OpenSearch Benchmark` -> `you can launch a new cluster to be used by OpenSearch Benchmark`